sphinx_design/compiled/sd_tabs.js

```diff
@@ -1,7 +1,19 @@
 var sd_labels_by_text = {};
 
+const storageKey = '&{storage_key}';
+// The default value for paramKey is the string 'None'. That value indicates
+// not to set a URL search parameter.
+const paramKey = '&{param_key}';
```

Comment:
I'm a bit confused about the UX if people have multiple groups of synced tabs throughout their site. In that case would it just store the state of the synced tab group that was last-clicked?

Reply:
That sounds correct--if one Sphinx project has a set of synchronized tabs with `python`, `java`, and `c` and another set with `lua` and `shell`, or `windows` and `linux` for that matter, then whatever is clicked last is set in the storage key.  We'd need add another option for the tab and store the data so folks could specify the `python-java-c` sets differently from the `windows-linux` sets.

I was thinking that the typical use is for a Sphinx project to use the tabs for programming language, platform, or user interface.  If you feel that isn't featureful enough and want to add an option, LMK and I'll see what I can do.

Reply:
I typed more about the UX, though I might have poisoned the problem.